src/js/pages/GranthIndex/components/AmritKeertanIndex.tsx

```diff
@@ -11,7 +11,7 @@ interface IAmritKeertanProps {
 export const AmritKeertanIndex: React.FC<IAmritKeertanProps> = ({ data }) => {
 
   return (
-    <div>
+    <div className="granthIndex">
```

Comment:
if this class just for the selector? if yes then prefer if `data-cy` like in HeaderMenu.